docs/content/new-tutorial/intro-to-assets.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: "Introduction to the Dagster Tutorial and Assets | Dagster Docs"
+description: "Learn about the objectives for the Dagster tutorial and about assets"
+noindex: True
+---
+
+# Introduction to Dagster
+
+Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports.
+
+In this tutorial, you will analyze activity on a popular news aggregation website. Similar to your data pipelines, you will fetch data from the website, clean it up, and build a report that summarizes some findings. You'll then tell Dagster to occassionally update the data and the report, which Dagster calls _assets_.
+
+---
+
+## Introduction to Software-defined assets
+
+Before building a Dagster project, you'll first learn about Dagster's core concept: the Software-defined asset (SDA).
+
+An asset is an object in persistent storage that captures some understanding of the world. Assets can be any type of object, such as:
+
+- A database table or view
+- A file, such as in your local machine or blob storage like Amazon S3
+- A machine learning model
+
+A Software-defined asset is Dagster's concept that couples an asset to the function and upstream assets that are used to produce its contents.
+
+If you have an existing data pipeline, you likely already have assets.
+
+### Goals
+
+By the end of this section, you will:
+
+- Know what assets are
+- Know the power of Software-defined assets
+- Know when to use task-based orchestration with assets
+- Start to think about what parts of your current pipeline are assets
+
+### Parts of a Software-defined asset
+
+Software-defined assets wrap objects to enable data engineers to do more with them.
+
+As a Dagster user, you're able to build assets in code. In the sample asset below, the data frame returned is an asset and Dagster recognizes that the function is an SDA 
+
+```
+@asset
+def hackernews_topstories(hackernews_topstory_ids):
+    results = []
+    for item_id in hackernews_topstory_ids:
+        item = requests.get(
+            f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+        ).json()
+        results.append(item)
+
+    return pd.DataFrame(results)
+```
+
+### Building a DAG of assets
+
+Defining multiple assets creates a DAG (directed acyclic graph) of what assets depend on other assets. This DAG, viewable in Dagster's web-based UI, helps you to:
```

Comment:
there's a slight leap here which is that we talk about a single asset, and then we talk about DAGs. Is it worth it to call out how dependencies are actually defined between assets?

Reply:
@elementljarredc D'you think it's enough to add something like:

> Dependencies can be upstream or downstream of an asset. We'll discuss defining asset dependencies later in the tutorial.

I'm hesitant to suggest going too in-depth here, as there's a whole section dedicated to dependencies. I wouldn't want to overload people in the intro.

Reply:
Rolling up a reply to both this and [this](https://github.com/dagster-io/dagster/pull/11614/files#r1092617219) in one:

Yeah, this used to be a more thorough explanation of assets and covered the points you mentioned, but the feedback was that the page was getting long into the conceptual for a first page of a tutorial.

I can see how I probably over-corrected, though. Maybe an explicit line about lineage?